lib/rules/validate-jsdoc/enforce-existence.js

```diff
@@ -26,18 +27,27 @@ enforceExistence.configure = function(options) {
         'jsDoc.enforceExistence rule was not configured properly'
     );
 
+    var optionsToPolicyMap = [
```

Comment:
Sure, sorry about that. I use WebStorm EAP, looks like editor config settings are not picked up properly. Also jscs behaves funny. I think in .jscsrc we need to change `"plugins": ["jscs-jsdoc"]` to be `"plugins": ["../jscs-jsdoc"]`. That way both WebStorm integration and `npm run lint` work well. I'll try to update in this feature and see if build will pass.